battle.go

```diff
@@ -228,3 +248,13 @@ type FightTurn struct {
 func (turn FightTurn) Priority() int {
 	return 0
 }
+
+// Indicates that:
+//      in a multibattle, the Team this Agent belongs to will forfeit.
+//      in a wild Pokemon battle, the Agent will attempt to escape.
+// Battles against NPC Trainers cannot be forfeit or escaped.
+type RunTurn struct{}
+
+func (turn RunTurn) Priority() int {
+	return 7 // outside the legal range for any FightTurn
```

Comment:
Why 7?

Reply:
6 is the maximum priority for a FightTurn, I'll change the comment to be clearer

Reply:
actually I'll just go make a constant `MAX_PRIORITY` and return that +1

Reply:
I added these constants in #43

Reply:
since we discussed that the priority of turn is a separate concept from the priority of a move, I'm thinking that `RunTurn`'s priority should be 3

`RunTurn` > `SwitchTurn` > `ItemTurn` > `FightTurn` is what I'm thinking